src/components/common/state-machine/state-machine.ts

```diff
@@ -725,6 +725,9 @@ const authStateMachine = createMachine(
           [USER_JOURNEY_EVENTS.EMAIL_SECURITY_CODES_CODE_VERIFIED]: [
             PATH_NAMES.GET_SECURITY_CODES,
           ],
+          [USER_JOURNEY_EVENTS.PASSWORD_RESET_INTERVENTION]: [
```

Comment:
Could you add a test for this? I think you should be able to modify one of the existing ones in the reset password integration tests. Also, would we not need to modify the middleware config too? (the true / true thing we were talking about yesterday?) If easier to chat through on a call let me know

Reply:
Ah sorry I was thinking of the other ticket we were on, so ignore the middleware comment. I still would want an overall test for this